gradle.properties

```
@@ -36,3 +36,8 @@ stdlibAuthVersion=1.0.0-SNAPSHOT
 stdlibOAuth2Version=1.0.0-SNAPSHOT
 stdlibLdapVersion=1.0.0-SNAPSHOT
 stdlibCryptoVersion=1.0.0-SNAPSHOT
+stdlibJavaarraysVersion=0.9.0-SNAPSHOT
```

Shall we rename as follows?
```suggestion
stdlibJavaArraysVersion=0.9.0-SNAPSHOT
```

---

+1. Fixed in 0e96c07